FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -473,23 +480,23 @@ def _getCentralCfg(self, installCfg):
           users = centralCfg['Registry']['Groups'][group].getOption(  # pylint: disable=unsubscriptable-object
               'Users', [])
           if adminUserName not in users:
-            centralCfg['Registry']['Groups'][group].appendToOption(  # pylint: disable=unsubscriptable-object
-                'Users', ', %s' % adminUserName)
-          if not centralCfg['Registry']['Groups'][group].isOption(  # pylint: disable=unsubscriptable-object
-                  'Properties'):
-            centralCfg['Registry']['Groups'][group].addKey(  # pylint: disable=unsubscriptable-object
-                'Properties', '', '')
-
-        properties = centralCfg['Registry']['Groups'][adminGroupName].getOption(  # pylint: disable=unsubscriptable-object
-            'Properties', [])
+            centralCfg['Registry']['Groups'][group].appendToOption(
+                'Users', ', %s' % adminUserName)  # pylint: disable=unsubscriptable-object
```

Comment:
pylint is failing because you moved the ignore away from line 483, same for some other lines.
```
FrameworkSystem/Client/ComponentInstaller.py:483: [E1136(unsubscriptable-object), ComponentInstaller._getCentralCfg] Value 'centralCfg['Registry']' is unsubscriptable
FrameworkSystem/Client/ComponentInstaller.py:485: [E1136(unsubscriptable-object), ComponentInstaller._getCentralCfg] Value 'centralCfg['Registry']' is unsubscriptable
FrameworkSystem/Client/ComponentInstaller.py:487: [E1136(unsubscriptable-object), ComponentInstaller._getCentralCfg] Value 'centralCfg['Registry']' is unsubscriptable
FrameworkSystem/Client/ComponentInstaller.py:490: [E1136(unsubscriptable-object), ComponentInstaller._getCentralCfg] Value 'centralCfg['Registry']' is unsubscriptable
FrameworkSystem/Client/ComponentInstaller.py:495: [E1136(unsubscriptable-object), ComponentInstaller._getCentralCfg] Value 'centralCfg['Registry']' is unsubscriptable
FrameworkSystem/Client/ComponentInstaller.py:498: [E1136(unsubscriptable-object), ComponentInstaller._getCentralCfg] Value 'centralCfg['Registry']' is unsubscriptable```

Reply:
I moved back. Thanks!